traffic_control/clients/python/trafficops/tosession.py

```diff
@@ -1522,6 +1522,38 @@ def delete_multiple_servers_capabilities(self, data=None):
 		:raises: Union[LoginError, OperationError]
 		"""
 
+	#
+	# Server_server_capabilities
+	#
+	@api_request('get', 'server_server_capabilities', ('3.0', '4.0', '4.1', '5.0'))
+	def get_server_server_capabilities(self, query_params=none):
```

Comment:
query_params=none: check language syntax: -> None

Reply:
Updated to None.